mkdocs/docs/tutorials/testing-patterns/conformance.md

```diff
@@ -20,8 +20,64 @@ template: main.html
     
 Please follow steps 1, 2, and 3 of the [quick start tutorial](../../../getting-started/quick-start/#1-create-kubernetes-cluster).
 
-
 ## 4. Create app/ML model version
+=== "Istio"
```

Comment:
This says "Deploy  a Knative app" while it should probably say "deploy an app?"

Reply:
done